Pipfile

```diff
@@ -0,0 +1,32 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+flake8 = "*"
+flake8-bandit = "*"
+flake8-bugbear = "*"
+flake8-import-order = "*"
+flake8-string-format = "*"
+flake8-tidy-imports = "*"
+mccabe = "*"
+pep8-naming = "*"
+coverage = "*"
+unittest-xml-reporting = "*"
+
+[packages]
+django = ">=2.2"
+django-crispy-forms = "*"
+django-environ = "*"
+django-filter = "*"
+django-hosts = "*"
+djangorestframework = "*"
+djangorestframework-bulk = "*"
+uwsgi = "*"
```

Comment:
uWSGI was installed as an extra because it's compiled in C and this takes up a bunch of extra dependencies. `psycopg2-binary` is different in this regard because it provides wheels: https://pypi.org/project/psycopg2-binary/#files.

For local development you don't need uWSGI, unless you want to test something super specific. At least we don't need it now (we might need it later when we're using things built into uWSGI itself, like [setting metrics for the uWSGI metrics server](https://uwsgi-docs.readthedocs.io/en/latest/Metrics.html)).

I don't know if Pipenv supports extras, but if it doesn't, then this should be removed from here altogether and only installed in the production Dockerfiles.

Reply:
According to both `pip` and `pipenv`, `uwsgi` does not have a single dependency.

Which sounds odd to me, honestly.

Anyway, Pipenv does not support extras.